lib/util.js

```diff
@@ -403,6 +403,108 @@ module.exports = {
     };
   },
 
+  /**
+   * Generates object containing objects of tags and their requests
+   * @param {object} spec - openAPI spec object
+   * @returns {object} returns an object containing objects of tags and their requests
+   */
+  /* eslint-disable max-depth */
+  generateFolderObjectFromTags: function(spec) {
+    var globalTags = spec.tags,
+      localTags = [],
+      globalTag = '',
+      paths = spec.paths,
+      currentPathObject = {},
+      operationObject = {},
+      folderObj = {
+        withTags: {},
+        withoutTags: {
+          requests: []
+        }
+      },
+      collectionVariables = {},
+      commonParams = '',
+      summary = '',
+      pathLevelServers = '',
+      name = '',
+      description = '';
+
+    // checking if tags are defined at the root level
+    if (globalTags.length > 0) {
```

Comment:
what exactly are we trying to check here?
if `globalTags` is an object, why are we checking for length?

Reply:
`globalTags` is an array.